addons/hr_timesheet/project_timesheet.py

```diff
@@ -13,6 +13,47 @@ class Project(models.Model):
         help="Choosing a sub-tasks project will both enable sub-tasks and set their default project (possibly the project itself)")
     allow_timesheets = fields.Boolean("Allow timesheets", default=True)
 
+    @api.multi
+    def action_view_timesheet(self):
```

Comment:
Since the spec change at least twice, a clean should be done.
Initially, 2 differents modules (having no dependence in commun) could provide the Plan. Now, project_timesheet_forecast_sale depends on sale_timesheet, so actually we can move all Plan stuff (js, controller) from hr_timesheet into sale_timesheet directly.
Do you copy ? over.